src/components/notification/channel/teams.ts

```diff
@@ -23,78 +23,118 @@
  **********************************************************************************/
 
 import axios from 'axios'
+import { NotificationMessage } from '../../../interfaces/notification'
 import { TeamsData } from './../../../interfaces/data'
-import { AxiosResponseWithExtraData } from '../../../interfaces/request'
-import getIp from '../../../utils/ip'
-import { hostname } from 'os'
-import { publicIpAddress } from '../../../utils/public-ip'
 
-export const sendTeams = async (data: TeamsData) => {
-  try {
-    const notifType = data.body.probeState === 'UP' ? 'RECOVERY' : 'INCIDENT'
-    const notifColor = data.body.probeState === 'UP' ? '8CC152' : 'DF202E'
+export const sendTeams = async (
+  data: TeamsData,
+  message: NotificationMessage
+) => {
+  if (message.meta.type === 'start' || message.meta.type === 'termination') {
```

Comment:
Can we use switch case?

Reply:
yes, will update soon

Reply:
updated in [c4354a6](https://github.com/hyperjumptech/monika/pull/354/commits/c4354a6199f3fd9f804b832b37d2407b158e6cbb)